content/docs/rfcs/62/README.md

```diff
@@ -0,0 +1,1077 @@
+| 14 | verification_status | `int64` | The verification status of the contact |
+| 15 | trust_status | `int64` | The trust status of the contact |
+| 16 | contact_request_local_state | `int64` | The local contact request state |
+| 17 | contact_request_local_clock | `int64` | The local contact request clock |
+| 18 | contact_request_remote_state | `int64` | The remote contact request state |
+| 19 | contact_request_remote_clock | `int64` | The remote contact request clock |
+
+### SyncInstallationPublicChat
+
+The node uses `SyncInstallationPublicChat` message to synchronize in a best-effort the public chats to other devices.
+
+```protobuf
+message SyncInstallationPublicChat {
+  uint64 clock = 1;
+  string id = 2;
+}
+```
+
+#### Payload
+
+| Field | Name | Type | Description |
+| ----- | ---- | ---- | ---- |
+| 1 | clock | `uint64` | clock value of the chat | 
+| 2 | id | `string` | id of the chat synced |
+
+### SyncPairInstallation
+
+The node uses `PairInstallation` messages to propagate information about a device to its paired devices.
+
+```protobuf
+message SyncPairInstallation {
+  uint64 clock = 1;
+  string installation_id = 2;
+  string device_type = 3;
+  string name = 4;
+  // following fields used for local pairing
+  uint32 version = 5;
+}
+```
+
+#### Payload
+
+| Field | Name | Type | Description |
+| ----- | ---- | ---- | ---- |
+| 1 | clock | `uint64` | clock value of the chat | 
+| 2| installation_id | `string` | A randomly generated id that identifies this device |
+| 3 | device_type | `string` | The OS of the device `ios`,`android` or `desktop` |
+| 4 | name | `string` | The self-assigned name of the device |
+
+### ChatIdentity
+
+`ChatIdentity` represents the user defined identity associated with their public chat key.
+
+```protobuf
+message ChatIdentity {
+  uint64 clock = 1;
+  string ens_name = 2;
+  map<string, IdentityImage> images = 3;
+  string display_name = 4;
+  string description = 5;
+  string color = 6;
+  string emoji = 7;
+  repeated SocialLink social_links = 8;
+  // first known message timestamp in seconds (valid only for community chats for now)
+  // 0 - unknown
+  // 1 - no messages
+  uint32 first_message_timestamp = 9;
+}
+```
+
+#### Payload
+
+| Field | Name | Type | Description |
+| ----- | ---- | ---- | ---- |
+| 1 | clock | `uint64` | Clock value of the message | 
+| 2| ens_name | `string` | A valid ENS associated with the chat key |
+| 3 | images | `map<string, IdentityImage>` | Image data associated with the chat key |
+| 4 | display_name | `string` | The self-assigned display_name of the chat key |
+| 5 | description | `string` | The description of the chat |
+| 6 | color | `string` | The color of the chat |
+| 7 | emoji | `string` | The emoji of the chat |
+| 8 | social_links | `array<SocialLink>` | A list of links to social platforms |
+| 9 | first_message_timestamp | `uint32` | First known message timestamp in seconds |
+
+### CommunityDescription
```

Comment:
This has been covered in https://rfc.vac.dev/spec/56/#wire-format, maybe link there